src/Components/Pages/NameAndDescriptionPage.js

```diff
@@ -29,13 +38,15 @@ const SetNameAndDescriptionPage = ({
   );
 };
 
-SetNameAndDescriptionPage.propTypes = {
+NameAndDescriptionPage.propTypes = {
+  mode: PropTypes.oneOf(['create', 'edit']).isRequired,
   name: PropTypes.string.isRequired,
+  activationKey: PropTypes.object.isRequired,
```

Comment:
Polish: Remove this prop type definition - activation key is not a prop for this component